src/preact/base-element.js

```diff
@@ -251,16 +259,14 @@ export class PreactBaseElement extends AMP.BaseElement {
     const Ctor = this.constructor;
 
     this.observer = new MutationObserver(this.checkMutations_.bind(this));
-    const childrenInit = Ctor['children'] ? CHILDREN_MUTATION_INIT : null;
-    const passthroughInit =
-      Ctor['passthrough'] || Ctor['passthroughNonEmpty']
-        ? PASSTHROUGH_MUTATION_INIT
+    const childrenInit =
+      Ctor['props'] && Ctor['props']['children']
+        ? CHILDREN_MUTATION_INIT
```

Comment:
Could use a second opinion here -- `CHILDREN_MUTATION_INIT` now always contains `characterData: true` even if none of the given props use `passthrough` or `passthroughNonEmpty`. Would it be better to ascertain concretely by looking for these in the props first?

Reply:
Done, thanks!